### PR TITLE
ceph.spec: drop support for el7 rpm builds

### PR DESCRIPTION
**Pull request by ideepika**
Signed-off-by: Deepika Upadhyay <dupadhya@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by ideepika** [comment]
shaman builds: https://shaman.ceph.com/builds/ceph/drop-el7/

File: ceph.spec.in [hunk 1]
```diff
@@ -134,20 +131,10 @@ BuildRequires:	checkpolicy
...
 BuildRequires:	gperf
-%if 0%{?rhel} == 7
-BuildRequires:  cmake3 > 3.5
-%else
 BuildRequires:  cmake > 3.5
-%endif
 BuildRequires:	cryptsetup
 BuildRequires:	fuse-devel
-%if 0%{?rhel} == 7
-# devtoolset offers newer make and valgrind-devel, but the old ones are good
```
Comment (tchaikov) [inline ceph.spec.in:145]: could you update `install-deps.sh` to drop the dts (devtoolset ) related bits?

**Comment by tchaikov** [comment]
please s|el7/rhel7|el7| in the title.

File: ceph.spec.in (hunk 1 above)
Comment (ideepika) [reply ceph.spec.in:145]: sure!

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2020-06-25_08:00:18-rados-wip-kefu-testing-2020-06-25-1439-distro-basic-smithi/

**Pull request merged**